src/mqtt/mqtt_test.c

```diff
@@ -1003,7 +989,7 @@ TEST( MqttTest, MQTT_Connect_LWT )
  */
 TEST( MqttTest, MQTT_ProcessLoop_KeepAlive )
 {
-    uint32_t connectPacketTime = context.lastPacketTime;
+    uint32_t connectPacketTime = context.lastPacketTxTime;
```

Comment:
Is this update for new coreMQTT version? If it is, suggest we use MQTT version to support previous LTS version.